acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/ContentSteps.java

```diff
@@ -266,30 +268,34 @@ public void iCanSeeAListOfPublicationsFromTheTimeSeries() throws Throwable {
             is(expectedPublicationTitles));
     }
 
-    // Scenario: Title validation ========================================================================
-    @When("^I populate the title with text which is too long$")
-    public void whenIPopulateTheTitleWithTextWhichIsTooLong() throws Throwable {
+    // Scenario: Title and Summary validation ========================================================================
+    @When("^I populate the (title|summary) with text which is ([0-9]+) characters long$")
+    public void whenIPopulateTheXWithLongText(String fieldName, int length) throws Throwable {
+
         StringBuilder sb = new StringBuilder();
-        for (int i=0; i < 251; i++) {
+        for (int i=0; i < length; i++) {
             sb.append(i % 10);
         }
         String longString = sb.toString();
-        assertThat("String should be > 250 characters long", longString.length(), greaterThan(250));
 
-        contentPage.populatePublicationTitle(longString);
+        if (fieldName.equals("title")) {
+            contentPage.populatePublicationTitle(longString);
+        } else if (fieldName.equals("summary")) {
+            contentPage.populatePublicationSummary(longString);
+        }
```

Comment:
It may be just me, but I'd split this into two, separate method and eliminate the if/else logic, while moving the string generation into a common helper/utility method.

Admittedly, right now it's a rather simple case, and in tests it may have even less impact than in production code, but an initially innocent looking if/else logic is prone to quickly become difficult to maintain as we may be tempted to add cases to it; in general, the less conditional logic the easier to read and maintain (and test, in case of production code) the code is so it's best to avoid it if we can.

Reply:
We all appreciate the situation here and agreed that just adding a clarifying comment will suffice for now.